libs/blocks/global-navigation/global-navigation.js

```diff
@@ -113,15 +110,15 @@ class Gnav {
         loadBlock('./blocks/appLauncher/appLauncher.js'),
         loadBlock('./blocks/profile/profile.js'),
         loadBlock('./blocks/search/gnav-search.js'),
-        loadStyles('profile/profile.css'),
-        loadStyles('navMenu/menu.css'),
-        loadStyles('search/gnav-search.css'),
+        loadStyles('./blocks/profile/profile.css'),
+        loadStyles('./blocks/navMenu/menu.css'),
+        loadStyles('./blocks/search/gnav-search.css'),
       ]);
       this.menuControls = new MenuControls(this.curtain);
       this.decorateMenu = decorateMenu;
       this.decorateLargeMenu = decorateLargeMenu;
       this.appLauncher = appLauncher;
-      this.Profile = Profile;
+      this.blocks.profile.Profile = Profile;
```

Comment:
Not a big fan of `profile.Profile` 😬 Could we switch it up a bit?

Reply:
Maybe `ProfileClass`? Open for suggestions!

It makes a bit more sense where it's used IMO
```js
const initProfileMenu = async ({ blockEl, ProfileClass, ...rest }) => {
  const placeholders = await getPlaceholders(['profile-button', 'sign-out', 'view-account', 'manage-teams', 'manage-enterprise']);
  const profile = new ProfileClass({
    localMenu: blockEl.querySelector('h5')?.parentElement,
    placeholders,
    ...rest,
  });
  return profile;
};
```